libraries/Microsoft.Bot.Builder/TurnContext.cs

```diff
@@ -51,6 +53,16 @@ public TurnContext(BotAdapter adapter, Activity activity)
         /// <value>The services registered on this context object.</value>
         public TurnContextStateCollection TurnState { get; } = new TurnContextStateCollection();
 
+        /// <summary>
```

Comment:
Has this new property been validated by Tom, Steve, etc? In general we are using the activity locale in our code to drive localization, do we really need a new context property?

Reply:
Yes, this new property is based on Tom's opinion. A context property can help the locale to be passed to adaptive-expressions